src/components/TestimonialSection/testimonial.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import clsx from 'clsx';
+import Heading from '@theme/Heading';
+import styles from './TestimonialSection.module.css';
+
+const Testimonials = [
+  {
+    name: "J. COLE",
```

Comment:
Get real names for these testimonials. Request who and who should be here from Sir Wisdom